packages/react/src/components/Tile/next/Tile.js

```diff
@@ -436,6 +437,12 @@ export function ExpandableTile({
     setPrevTilePadding(tilePadding);
   }
 
+  let interactiveContent;
+
+  if (belowTheFold.current) {
+    interactiveContent = getInteractiveContent(belowTheFold.current);
+  }
+
```

Comment:
@abbeyhrt I think there may need to be an effect used here because what can happen is that `belowTheFold.current` will be `undefined` on the first render. It isn't until later renders that this value will potentially be defined.

Here's a demo I made to try and isolate this, let me know if it relates or not!
https://codesandbox.io/s/spring-haze-ol99tt?file=/src/App.js